File: lc1365.js
var smallerNumbersThanCurrent = (nums)=> {
    let n = nums.length;
    let ans = [];
    for (let i=0;i<n;i++){
        let cnt = 0;
        for(let j=0;j<n;j++){
            cnt = (nums[j]<nums[i])?cnt+1:cnt+0;
        }
        ans.push(cnt);
    }
    return ans;
};

File: py/lc1365.py
def smallerNumbersThanCurrent(nums):
    n = len(nums)
    ans = []
    for i in range(n):
        cnt = 0
        for j in range(n):
            if nums[j]<nums[i]:
                cnt = cnt + 1
        ans.append(cnt)
    return ans

nums = [8,1,2,2,3]
print(smallerNumbersThanCurrent(nums))
